core/src/main/java/co/aikar/commands/BaseCommand.java

```diff
@@ -518,6 +519,10 @@ public boolean canExecute(CommandIssuer issuer, RegisteredCommand<?> cmd) {
                     continue;
                 }
 
+                if(value.method.isAnnotationPresent(Private.class)){
```

Comment:
Please look up the annotation in the RegisteredCommand ctor, and store a boolean there, and use the annotations class like everything else does